cos_registration_server/openapi.yaml

```diff
@@ -0,0 +1,1267 @@
+openapi: 3.0.3
```

Comment:
This file is automatically generated right?
Any reason to commit it other than the test in CI and link in readme? If not I'd seek to generate it in a CI job and push it to a separate branch on the repo, e.g. `doc/api`, if it needs to be referenced.

Reply:
This is another solution, but I thought that since we won't change that often we don't need such an automation.
Plus, in case you want to test your file, you cannot really while here you can just point to your dev branch.